docs/Getting.md

```diff
@@ -1,11 +1,23 @@
 **Cromwell Releases**
 
-The preferred way of getting Cromwell releases is at the [GitHub Releases](https://github.com/broadinstitute/cromwell/releases/latest) page. 
+Cromwell releases are available at the [GitHub Releases](https://github.com/broadinstitute/cromwell/releases/latest) page. 
 You are strongly encouraged to use the latest release of Cromwell whenever possible.
 
+Cromwell is distributed is a conda package on [conda-forge](https://conda-forge.org/).
+These instructions need to be followed for [installing the miniconda distribution](https://docs.conda.io/en/latest/miniconda.html) and 
+[activating the conda-forge channel](https://conda-forge.org/#about). After this Cromwell can be installed in the 
+base environment with `conda install cromwell` or a separate environment for Cromwell can be created with 
+`conda create -n cromwell cromwell`. If you are using Cromwell for bioinformatics workflows it is recommended you take 
+a look at [bioconda](http://bioconda.github.io)  as well. 
+The conda installation of Cromwell comes with a wrapper that eliminates the need for using 
+a `java -jar /path/to/jar` type of command. Conda also installs the required Java dependency 
```

Comment:
Maybe say what you _do_ need to do, instead of what you don't?

Eg: "to run Cromwell now, you just need to run `cromwell run ...` or something similar?

Reply:
Much better indeed. Done.